domain/src/main/java/org/oppia/android/domain/spotlight/SpotlightStateController.kt

```diff
@@ -158,12 +129,13 @@ class SpotlightStateController @Inject constructor(
     val cacheStore = if (profileId in cacheStoreMap) {
       cacheStoreMap[profileId]!!
     } else {
-      val cacheStore =
+      val cacheStore = cacheStoreMap.getOrPut(profileId) {
```

Comment:
Now that this is using getOrPut, you can just return it. The code on lines 139 and 140 are no longer needed.

Reply:
done